### PR TITLE
chore: enable unit tests

### PR DESCRIPTION
**Pull request by peternhale**
@W-10023763@

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "tar-fs": "^2.1.1"
   },
   "devDependencies": {
+    "@oclif/dev-cli": "^1.26.0",
```
Comment (mdonnalley) [inline package.json:22]: You don't need this since since you already have oclif@2 as a dev dependency

File: test/commands/update.nut.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import * as qq from 'qqjs'
 
 const skipIfWindows = process.platform === 'win32' ? it.skip : it
 
-describe('update', () => {
+describe.skip('update', () => {
```
Comment (mdonnalley) [inline test/commands/update.nut.ts:7]: why was this file renamed to `update.nut.ts`?

**Review by mdonnalley — CHANGES_REQUESTED** [review]

File: test/commands/update.nut.ts (hunk 2 above)
Comment (peternhale) [reply test/commands/update.nut.ts:7]: Because it really is a nut test

Comment (mdonnalley) [reply test/commands/update.nut.ts:7]: But oclif doesn't have any concept of NUTs - and looking at the package.json it won't ever be run since it doesn't match the regex

Comment (peternhale) [reply test/commands/update.nut.ts:7]: I left in place just in case we add nuts to oclif. I you prefer I can delete this file.

**Review by mdonnalley — APPROVED** [review]
